packages/extension-api/src/extension-api.d.ts

```diff
@@ -2424,4 +2424,47 @@ declare module '@podman-desktop/api' {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     export function setValue(key: string, value: any, scope?: 'onboarding'): void;
   }
+
+  export interface CliToolOptions {
+    id: string;
+    name: string;
+    displayName: string;
+    markdownDescription: string;
+    images: ProviderImages;
+    location?: string;
+    version?: string;
+    detection?: {
+      versionOptions: string[]; // run command by name using these options
+      versionParser: (string) => string;
+    };
+  }
+
+  export type CliToolState =
+    | 'setup-needed'
+    | 'detection-pending'
+    | 'detecting'
+    | 'installation-pending'
+    | 'installing'
+    | 'update-pending'
+    | 'updating'
+    | 'installed'
+    | 'installed-unknown'
+    | 'uninstall-pending'
+    | 'uninstalling'
+    | 'unknown';
+
+  export interface CliTool extends CliToolOptions, Disposable {
```

Comment:
why CLiTool is extending CliToolOptions ?

Reply:
because the same properties should be on CliTool anyway, I can copy/paste it if that is better.